com.obsproject.Studio.Plugin.Vaapi.yml

```diff
@@ -0,0 +1,29 @@
+id: com.obsproject.Studio.Plugin.Vaapi
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/Vaapi
+modules:
+  - name: vaapi
+    buildsystem: meson
+    config-opts:
+      - --buildtype=release
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ../${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak
+        ${FLATPAK_ID}
+    sources:
+      - type: git
+        url: https://github.com/fzwoch/obs-vaapi.git
+        tag: 0.1.0
+        commit: 4eac24b412f12e9bbeff0bea934d16ab83e69c57
+        x-checker-data:
+            type: git
+            tag-pattern: ^v([\d.]+)
```

Comment:
Give the tag above, this pattern is unlikely to match.

Reply:
With the recent https://github.com/flathub/flathub/pull/3862/commits/c2b6271d75027d25c6509a300b2660afa2545042, i have confirmed locally that flatpak-external-data-checker returns no errors